model_compression_toolkit/common/mixed_precision/mixed_precision_quantization_config.py

```diff
@@ -28,8 +28,8 @@ def __init__(self,
                  weights_n_bits: List[int] = None,
                  compute_distance_fn: Callable = compute_mse,
                  distance_weighting_method: Callable = get_average_weights,
-                 num_of_images: int = 32):
-
+                 num_of_images: int = 32,
+                 configuration_overwrite: List[int] = None):
         """
```

Comment:
Are you sure you want to push this? It's for debug purposes

Reply:
yes